frontend/appflowy_flutter/integration_test/desktop/uncategorized/code_block_language_selector_test.dart

```diff
@@ -45,7 +44,7 @@ void main() {
     await onKey(tester, LogicalKeyboardKey.enter);
 
     final editorState = tester.editor.getCurrentEditorState();
-    final language =
+    var language =
```

Comment:
You can replace the 'language' with `CodeBlockKeys.language`.

Reply:
`CodeBlockKeys.language` is a global const value, and the `language` variable here is a local variable, I'm using 'var' here for convenience in case I need to reuse it later, so I don't have to recreate the object.

Reply:
We don't use `var` as it is ambiguous, declare the type properly.

Reply:
This's a reasonable explanation, I'll declare it as `String`

Reply:
@asjqkkkk I was referring to the attribute key, not the variable name.

Reply:
I've changed `attributes['language']` to `attributes[CodeBlockKeys.language]`